### PR TITLE
refactor: rename references of database-viewer to viewer

### PR DESCRIPTION
**Pull request by bhajneet**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,121 +1,100 @@
-
-# Shabad OS Database Viewer
-
-<div align="center">
-  <img src="./repologo.png">
-</div>
-<br/>
-<div align="center">
-
-Work in-progress: An online viewer of the bani database. Authenticated by GitHub for suggesting changes. Shown as pull requests to our [Database](https://github.com/ShabadOS/database) repo.
```
Comment (Harjot1Singh) [inline README.md:10]: `Authenticated by GitHub for suggesting changes. Shown as pull requests to our [Database](https://github.com/ShabadOS/database) repo.` Some of this blurb i thought was actually quite useful.

Comment (bhajneet) [reply README.md:10]: It's not authenticated in our website, it's just a link to open an issue

They are not shown as pull requests to our database repo

None of this was implemented

Comment (Harjot1Singh) [reply README.md:10]: That's fair enough - I think saying `opens issues in [link]` just so that readers understand the relationship between this and DB

Comment (bhajneet) [reply README.md:10]: I think the goal with the viewer is to decouple it from being a proofreading tool only, to being more of it's own entity the way presenter is

The presenter also allows one to submit a proofreading ticket. So it's not unique to the viewer, but I've still added this to the "features" section, idk if you noticed:

> Submit issue tickets for any line while proofreading

I don't think the reader of a readme cares where the ticket goes

Comment (Harjot1Singh) [reply README.md:10]: I'm ok with it being in features in that case. I'd still link to database issues, one may wish to have a quick browse out of piqued interest, no?

Comment (bhajneet) [reply README.md:10]: Please [see updated readme](https://github.com/ShabadOS/viewer/blob/8eac917040145a7b299247787a3bfc2dfb27ccf7/README.md#features)

**Comment by bhajneet** [comment]
@Harjot1Singh The only concern I have is not knowing whether docker config.yml can be updated like this

**Comment by saihaj** [comment]
@bhajneet can you please update `package.json`

**Comment by bhajneet** [comment]
I must have forgot to push it yesterday. I thought I did and wrote that comment

**Comment by bhajneet** [comment]
@saihaj added files

File: README.md [hunk 2]
```diff
@@ -1,121 +1,100 @@
...
-[Sanpuran Steek Sri Sarabloh Granth Ji](https://drive.google.com/open?id=1-C8E3YYeAFIqacfe8Xy7scDFpTyn62_U) | [adelaidesangat.com](http://www.adelaidesangat.com/index.php/downloads/Gurbani-PDF/Sri-Sarabloh-Granth-Sahib/Complete-Sri-Sarbloh-Granth-Sahib-Ji-Steek.pdf/)
-[Sanpuran Steek Sri Sarabloh Granth Ji (Vol. 1)](https://drive.google.com/open?id=1-dbLAUCw2ME4gyBXS7QttId9WxDEkQnf)<br> [Sanpuran Steek Sri Sarabloh Granth Ji (Vol. 2)](https://drive.google.com/open?id=1ooERm2R6rwtedMTs1z21HakqtvHxMqjr) | Vol. 1: [panjabdigilib.org](http://panjabdigilib.org/webuser/searches/displayPage.jsp?ID=9210&page=1&CategoryID=1&Searched=)<br> Vol. 2: [panjabdigilib.org](http://panjabdigilib.org/webuser/searches/displayPage.jsp?ID=9209&page=1&CategoryID=1&Searched=)
```
Comment (saihaj) [inline README.md:113]: Where does this information go now? Is it in database docs too?

Comment (bhajneet) [reply README.md:113]: it's in [database readme already](https://github.com/ShabadOS/database#gurbani-and-panthic-compositions) and should be updated in the tutorial as well

**Review by saihaj — APPROVED** [review]

**Review by Harjot1Singh — APPROVED** [review]

**Review by Harjot1Singh — APPROVED** [review]

**Pull request merged**